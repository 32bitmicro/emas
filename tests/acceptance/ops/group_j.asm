UJ r0
JL r0
JE r0
JG r0
JZ r0
JM r0
JN r0
LJ r0
