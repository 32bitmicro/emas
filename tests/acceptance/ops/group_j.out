@ 0x0000 : 0xf000  /  111 100 0 000 000 000  /  61440
@ 0x0001 : 0xf040  /  111 100 0 001 000 000  /  61504
@ 0x0002 : 0xf080  /  111 100 0 010 000 000  /  61568
@ 0x0003 : 0xf0c0  /  111 100 0 011 000 000  /  61632
@ 0x0004 : 0xf100  /  111 100 0 100 000 000  /  61696
@ 0x0005 : 0xf140  /  111 100 0 101 000 000  /  61760
@ 0x0006 : 0xf180  /  111 100 0 110 000 000  /  61824
@ 0x0007 : 0xf1c0  /  111 100 0 111 000 000  /  61888
